guidance/models/llama_cpp/_llama_cpp.py

```diff
@@ -21,7 +21,7 @@
 class _LlamaBatchContext:
     def __init__(self, n_batch, n_ctx):
         self._llama_batch_free = llama_cpp.llama_batch_free
-        self.batch = llama_cpp.llama_batch_init(n_tokens=n_batch, embd=0, n_seq_max=n_ctx)
+        self.batch = llama_cpp.llama_batch_init(n_batch, 0, n_ctx)
```

Comment:
Per llama-cpp-python, I think the function signature here didn't change -- was there any issue with using named args instead of positional ones here? 

https://github.com/abetlen/llama-cpp-python/blob/8c71725d5345b2ba325e6d28209875057c5873e6/llama_cpp/llama_cpp.py#L1645

Reply:
In any case, fixing this seems to work -- just curious about why this changed! LGTM